core/src/test/resources/spaces/InParentheses.stat

```diff
@@ -0,0 +1,40 @@
+spaces.inParentheses = true
+maxColumn = 40
+<<< Spaces in parentheses
+function(a, b, c)
+>>>
+function( a, b, c )
+<<< no arg
+function()
+>>>
+function()
+<<< def
+def x(foo: Int) = 2
+>>>
+def x( foo: Int ) = 2
+<<< tuple
+def x = (1, 2)
+>>>
+def x = ( 1, 2 )
+<<< maxColumn (looks weird, but scalariform does the same)
```

Comment:
Interesting. Never seen that with scalariform.

Is it hard to let it produce this instead?
```
def x =
  ( 1,
    2,
    aaaaaaaaaaaaaawwwwsssssssssswa )
```

Are there any other flags I can set to let it produce either of:
```
def x = (
  1,
  2,
  aaaaaaaaaaaaaawwwwsssssssssswa )
```
```
def x = (
  1,
  2,
  aaaaaaaaaaaaaawwwwsssssssssswa
)
```

```
def x = (
  1, 2,
  aaaaaaaaaaaaaawwwwsssssssssswa
)
```
?